README.md

```diff
@@ -92,7 +92,5 @@ extensions for APM
   JavaScript SDKs with interfaces, type definitions and base classes.
 - [`@sentry/utils`](https://github.com/getsentry/sentry-javascript/tree/master/packages/utils): A set of helpers and
   utility functions useful for various SDKs.
-- [`@sentry/typescript`](https://github.com/getsentry/sentry-javascript/tree/master/packages/typescript): Shared
```

Comment:
Don’t think we want to remove this, we still rely on it for typescript configuration.